provisioning/RHCSA_RHCE_LAB/tasks/main.yml

```diff
@@ -11,33 +11,28 @@
     - libselinux-python
     - python-pip # for tzupdate
     - bash-completion
-    - bash-completion-extras
-    - dbus # for ansible nmcli http://docs.ansible.com/ansible/nmcli_module.html
   notify: update pip
 
-- name: install tzupdate
+- name: pip install tzupdate
   pip:
     name: tzupdate
 - name: set timezone based on geolocation
   # see https://github.com/cdown/tzupdate TODO CentOS GUI can be set to detect timezone, but how can we set this automated?
   shell: tzupdate
 
-
+# TODO labipa boots with GUI, install GUI on Server1 but boot in multi-user.target and on Server2 do not install GUI.
 - name: "Install 'Server With GUI' environment group"
```

Comment:
Sander wants the following:
- Server1 GUI installed, GUI not running
- Server2 GUI not installed
- labipa GUI installed and GUI running